src/jit/codegenarmarch.cpp

```diff
@@ -482,6 +482,45 @@ void CodeGen::genSetRegToIcon(regNumber reg, ssize_t val, var_types type, insFla
     instGen_Set_Reg_To_Imm(emitActualTypeSize(type), reg, val, flags);
 }
 
+//---------------------------------------------------------------------
+// genSetGSSecurityCookie: Set the "GS" security cookie in the prolog.
+//
+// Arguments:
+//     initReg        - register to use as scratch register
+//     pInitRegZeroed - OUT parameter. *pInitRegZeroed set to 'false' if 'initReg' is
+//                      not zero after this call.
```

Comment:
It seems like the semantics are/should be that it sets `*pInitRegZeroed` to false if and only if it sets `initReg` to a non-zero value.

Reply:
I have corrected the comments